.github/workflows/build-preview-environment.yml

```
@@ -0,0 +1,77 @@
+      image_name: workflows-dashboard
+      ref: ${{ github.event.pull_request.head.sha }}
+
+  deploy-preview:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Trigger workflow in another repo
+        uses: actions/github-script@v6
+        with:
+          github-token: ${{ secrets.GH_TOKEN }}
+          script: |
+            const refValue = ('${{ github.event_name }}' === 'workflow_dispatch') 
```

yamllint (1.29.0-1)

[error] trailing spaces

(trailing-spaces)